### PR TITLE
Feature/healthcheck

### PR DESCRIPTION
**Pull request by sb-keane**
Added a healthcheck endpoint. This branch comes from @sb-ebukaanene's docker branch so it includes those changes as well

### CODE REVIEW

File: src/main/java/gov/cms/ab2d/contracts/controller/HealthCheckController.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+@RestController
+@Slf4j
+public class HealthCheckController {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    @GetMapping("/health")
+    public ResponseEntity<Void> getHealth() {
+        try {
+            jdbcTemplate.query("SELECT 1", x -> {
```
Comment (sb-keane) [inline src/main/java/gov/cms/ab2d/contracts/controller/HealthCheckController.java:22]: I'm cool moving this to a service. As it was done as fast as possible, it's all KISS and no SOLID.

File: Jenkinsfile.deployment [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        jdk 'openjdk17'
+    }
+    
+
+    stages {
+   
+    
+        stage ('Build Libraries') {
+            steps {
+                withCredentials([usernamePassword(credentialsId: 'artifactoryuserpass', usernameVariable: 'ARTIFACTORY_USER', passwordVariable: 'ARTIFACTORY_PASSWORD')]) {
```
Comment (sb-dkniesteadt) [inline Jenkinsfile.deployment:21]: Probably need to build this more, test coverage, sonar ext

**Review by sb-dkniesteadt — APPROVED** [review]
Approved assuming comment is looked at.

File: Jenkinsfile.deployment (hunk 2 above)
Comment (sb-ebukaanene) [reply Jenkinsfile.deployment:21]: Done

**Review by sb-ebukaanene — APPROVED** [review]

**Pull request merged**